moler/device/juniper_ex.py

```diff
@@ -8,23 +8,62 @@
 __email__ = 'sylwester.golonka@nokia.com'
 
 from moler.device.junipergeneric import JuniperGeneric
+from moler.helpers import call_base_class_method_with_same_name, mark_to_call_base_class_method_with_same_name
 
 
+@call_base_class_method_with_same_name
 class JuniperEX(JuniperGeneric):
     """Juniperex device class."""
 
-    def _get_packages_for_state(self, state, observer):
-        available = {JuniperEX.cmds: [], JuniperEX.events: []}
-        if state == JuniperEX.unix_local or state == JuniperGeneric.proxy_pc:
-            available = {
-                JuniperEX.cmds: ['moler.cmd.unix'],
-                JuniperEX.events: ['moler.events.unix']}
-        elif state == JuniperGeneric.cli:
-            available = {
-                JuniperEX.cmds: ['moler.cmd.unix', 'moler.cmd.juniper.cli', 'moler.cmd.juniper_ex.cli'],
-                JuniperEX.events: ['moler.events.unix', 'moler.events.juniper', 'moler.events.juniper_ex']}
-        elif state == JuniperGeneric.configure:
-            available = {
-                JuniperEX.cmds: ['moler.events.unix', 'moler.cmd.juniper.configure', 'moler.cmd.juniper_ex.configure'],
-                JuniperEX.events: ['moler.events.unix', 'moler.events.juniper', 'moler.events.juniper_ex']}
-        return available[observer]
+    pass
+
+
+"""
+Example of device in yaml configuration file:
+    - with PROXY_PC:
+      JUNIPER_EX_PROXY_PC:
+        DEVICE_CLASS: moler.device.juniper_ex.JuniperEX
+        INITIAL_STATE: UNIX_LOCAL
+        CONNECTION_HOPS:
+          PROXY_PC:
+            CLI:
+              execute_command: ssh
+              command_params:
+                expected_prompt: "admin@switch>"
+                host: cli_host
+                login: cli_login
+                password: password
+                set_timeout: null
```

Comment:
Please remove all not necessary  parameters for e.g. set_timeout from example device dict.